lessons/intro/click/index.md

```diff
@@ -145,20 +157,25 @@ def mv(source, destination):
 ## Soubory
 
 Má-li uživatel zadat jméno souboru, nepoužívejte řetězce, ale speciální typ
-`click.File()`.
+[`click.File()`](http://click.pocoo.org/5/api/#click.File).
 Click za vás soubor automaticky otevře a zavře.
 Kromě toho podporuje unixovskou konvenci, že `-` znamená standardní
 vstup/výstup.
 
+Argument pro `File` je mód, ve kterém se soubor otevírá, podobně jako pro
+Pythonní funkci `open`: `'r'` pro čtení, `'w'` pro zápis.
+
 ```python
 @click.command()
 @click.argument('files', nargs=-1, type=click.File('r'))
 def cat(files):
+    """Print out the contents of the given files"""
     for file in files:
         print(file.read(), end='')
 ```
 
-Existuje i varianta `click.Path()`, která soubor neotvírá.
+Existuje i varianta [`click.Path()`](http://click.pocoo.org/5/api/#click.Path),
```

Comment:
Všecky přidané linky na dokumentaci clicku vedou na verzi 5. Je to záměr, nebo náhoda? Použil bych 6.